### PR TITLE
fix(drifts): double fields and services

### PR DESCRIPTION
**Pull request by LukaDevic**

### CODE REVIEW

File: docs/resources/connector.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ resource "fivetran_connector" "amplitude" {
 ### Optional
 
 - `auth` (Block List, Max: 1) (see [below for nested schema](#nestedblock--auth))
-- `config` (Block List, Max: 1) (see [below for nested schema](#nestedblock--config))
+- `config` (Block List) (see [below for nested schema](#nestedblock--config))
```
Comment (beevital) [inline docs/resources/connector.md:70]: Hmm, it should have Max: 1 attribute.

File: fivetran/connector_config_oas_service.go [hunk 2]
```diff
@@ -102,16 +97,15 @@ func getConnectorSchemaConfig(readonly bool, version int) *schema.Schema {
 
 	return &schema.Schema{
 		Type:     schema.TypeList,
-		Optional: true,
+		Optional: !readonly, // set to true when generating documentation for connector datasource
 		Computed: true,
-		MaxItems: 1,
```
Comment (beevital) [inline fivetran/connector_config_oas_service.go:107]: We should not delete this line!

File: docs/resources/connector.md (hunk 1 above)
Comment (LukaDevic) [reply docs/resources/connector.md:70]: Done. :)

File: fivetran/connector_config_oas_service.go (hunk 2 above)
Comment (LukaDevic) [reply fivetran/connector_config_oas_service.go:107]: Done. :)

File: docs/resources/connector.md (hunk 1 above)
Comment (beevital) [reply docs/resources/connector.md:70]: It still doesn't have Max: 1 attribute =)

**Review by beevital — APPROVED** [review]
LGTM

**Pull request merged**